protos/prototool.yaml

```diff
@@ -0,0 +1,71 @@
+# Protoc directives.
+protoc:
+  version: 3.10.0
+  includes:
+    - ../../../../ #FIXME: use $GOPATH rather than relative dir
+  allow_unused_imports: false
+
+# Lint directives.
+lint:
+  group: google
+  rules:
+    add:
+      - ENUM_NAMES_CAMEL_CASE
+      - ENUM_NAMES_CAPITALIZED
+    # remove:
+    #   - ENUM_NAMES_CAMEL_CASE
+  ignores:
+    - id: ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: ENUM_NAMES_CAMEL_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: ENUM_NAMES_CAPITALIZED
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: MESSAGE_FIELD_NAMES_LOWER_SNAKE_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+    - id: ENUM_FIELD_NAMES_UPPER_SNAKE_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+        - protos/protocol/core/Tron.proto
+    - id: ENUM_NAMES_CAMEL_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+        - protos/protocol/api/api.proto
+    - id: ENUM_NAMES_CAPITALIZED
+      files:
+        - protos/exchange/exchange_message.proto
+        - protos/protocol/api/api.proto
+        - protos/protocol/core/Tron.proto
+    - id: MESSAGE_FIELD_NAMES_LOWER_SNAKE_CASE
+      files:
+        - protos/exchange/exchange_message.proto
+        - protos/protocol/api/api.proto
+        - protos/protocol/core/Discover.proto
+        - protos/protocol/core/Tron.proto
+    - id: MESSAGE_NAMES_CAPITALIZED
+      files:
+        - protos/protocol/core/Tron.proto
+    - id: RPC_NAMES_CAPITALIZED
+      files:
+        - protos/protocol/api/api.proto
```

Comment:
this is from go-btfs-common -- probably need to make this config file simpler... or have a way to refer to go-btfs-common's makefile so there won't be more duplicated configs